src/Microsoft.DiaSymReader/Metadata/IMetadataImport.cs

```diff
@@ -26,37 +33,37 @@ internal unsafe interface IMetadataImport
         [PreserveSig]
         int EnumTypeDefs(
             ref void* enumHandle,
-            [Out]int* typeDefs, 
+            int* typeDefs,
```

Comment:
Are the Ref/In/Out attributes not needed for netstandard2.0?

Reply:
The attributes are rarely needed. In this case, the types are pointers so there is literally no impact.